NEWS.md

```diff
@@ -1,5 +1,9 @@
 # EpiNow2 (development version)
 
+## Documentation
+
+- The Getting Started vignette has been reworded in several places for clarity and a new section on how to specify different regional settings in `regional_epinow()` has been added. By @jamesmbaazam in # and reviewed by <REVIEWER>.
```

Comment:
```suggestion
- The Getting Started vignette has been reworded in several places for clarity and a new section on how to specify different regional settings in `regional_epinow()` has been added. By @jamesmbaazam in #1025 and reviewed by <REVIEWER>.
```

Reply:
I think the concensus is we are dropping the named review to improve PR flow